engine/src/foundation-legacy.cpp

```diff
@@ -1158,10 +1163,11 @@ bool MCNameClone(MCNameRef p_name, MCNameRef& r_new_name)
 	return true;
 }
 
-const char *MCNameGetCString(MCNameRef p_name)
-{
-	return MCStringGetCString(MCNameGetString(p_name));
-}
+// SN-2015-05-14: [[ MCStringGetCString Removal ]] End of the super-evil function
+//const char *MCNameGetCString(MCNameRef p_name)
+//{
+//	return MCStringGetCString(MCNameGetString(p_name));
+//}
 
 MCString MCNameGetOldString(MCNameRef p_name)
```

Comment:
The MCNameGetOldString method falls into the same category as MCNameGetCString and so needs to be removed at the same time.